frontend/package.json

```diff
@@ -22,7 +22,7 @@
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "cross-env react-scripts start",
```

Comment:
What's the purpose of adding this? Also doesn't work by default on my end.

Reply:
Whoops I didn't remove the first part of that command. It's to allow for injecting environment variables into the start scripts command.

Reply:
Fixed in [431347a](https://github.com/NickPriv/FryRank/pull/17/commits/431347a904429d6661bf52a7252525f3823c05c9)